src/Domain/Kernel.php

```diff
@@ -49,4 +55,20 @@ public static function getRequiredFiles(): array
             // '.gitignore',
         ];
     }
+
+    /**
+     * Returns the list of Insights required on root.
+     *
+     * @return array<string>
```

Comment:
<int, string>

Reply:
Are you sure ? 
There is no int in the array, and I was inspired by https://github.com/nunomaduro/phpinsights/blob/master/src/Domain/Kernel.php#L42

Reply:
Oh sorry my bad. Think I got confused with another codebase which did it like that recently :) 
I altso prefer it with string only 👍